test/forms/inputs/checkbox-group.test.js

```diff
@@ -125,3 +125,21 @@ test('CheckboxGroup passes down props to children', () => {
   expect(checkbox).not.toHaveClass('custom-group-class')
   expect(checkbox).toHaveClass('custom-input-class')
 })
+
+test('DropdownCheckboxGroup sets menu no longer active when clicked outside', async () => {
+  const user = userEvent.setup()
+
+  render(<WrappedCheckboxGroup value={['1']} useDropdown />)
+
+  const select = screen.getByRole('button')
+  await user.click(select)
+
+  expect(select.nextSibling).toHaveClass('options', 'is-active')
+
+  const fieldset = screen.getAllByRole('group').at(0)
```

Comment:
Once we remove the `role` attribute from `LabeledField`, does this still work? I know creating it as a `fieldset` automatically gives it a role of group, but is that actually available to select by here?